CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 #GUSINFO:MS CX Engineering,MS CX Engineering
-* @priyaayyar @chris-campbell9 @mulesoft/team-docs
+* @glenn-rodgers-sf @mulesoft/team-docs
```

Comment:
Hey @hannanelson, should you also include Marina as a codeowner?

Reply:
Yes, I was just coming back to add her 🤣